api/v1beta1/pachyderm_types.go

```diff
@@ -105,25 +105,13 @@ type EtcdOptions struct {
 	Service     *ServiceOverrides `json:"service,omitempty"`
 }
 
-// TODO: renove obsolete options such as NumShards, BlockCacheBytes, etc
 // PachdOptions allows the user to customize pachd
 type PachdOptions struct {
 	// Set an ID for the cluster deployment.
 	// Defaults to a random value if none is provided
 	ClusterID string `json:"clusterDeploymentID,omitempty"`
-	// Sets the maximum number of pachd nodes allowed in the cluster.
-	// Increasing this number blindly could lead to degraded performance.
-	// Default: 16
-	// +kubebuilder:default:=16
-	NumShards int32 `json:"numShards,omitempty"`
-	// Size of Pachd's in-memory cache for PFS file.
-	// Size is specified in bytes, with allowed SI suffixes (M, K, G, Mi, Ki, Gi, etc)
-	BlockCacheBytes string `json:"blockCacheBytes,omitempty"`
-	// Pachd memory request
-	// +kubebuilder:default:="1T"
-	MemoryRequest string `json:"memoryRequest,omitempty"`
 	// Resource requests and limits for Pachd
-	Resources corev1.ResourceRequirements `json:"resources,omitempty"`
+	Resources *corev1.ResourceRequirements `json:"resources,omitempty"`
```

Comment:
just building my understanding: are these files autogenerated? Are the removed fields determined by helm now?